chart/README.md

```diff
@@ -0,0 +1,501 @@
+    sail-operator-666f84b6f4-9hw4t   2/2     Running   0          43s   10.244.0.8   sail-control-plane   <none>           <none>
+    ```
+
+## Deploying Istio
+
+To deploy Istio, you must create the following resources:
+* In Kubernetes - `Istio`.
+* In OpenShift - `Istio` and `IstioCNI`.
+The `Istio` resource deploys and configures the Istio Control Plane, whereas the `IstioCNI` resource (in OpenShift) deploys and configures the Istio CNI plugin. You should create these resources in separate projects.
+
+### Create a namespace for Istio project.
+
+* Kubernetes
+
+    ```sh
+    $ kubectl create namespace istio-system
+    ```
+
+* OpenShift
+
+    ```sh
+    $ kubectl create namespace istio-system
+    $ kubectl create namespace istio-cni
+    ```
+
+### Create the Istio resource
+
+The `sail-operator` charts directory contains `samples` directory, which contains manifests that could be used for Istio deployment.
+
+* Kubernetes
+
+    ```sh
+    $ kubectl apply -f sail-operator/samples/istio-sample-kubernetes.yaml
+    ```
+
+* OpenShift
+
+    ```sh
+    $ kubectl apply -f sail-operator/samples/istio-sample-openshift.yaml
+    $ kubectl apply -f sail-operator/samples/istiocni-sample.yaml
+    ```
+
+**Note** - The Istio version deployment could be specified by modifying the `version` field within `Istio` and `IstioCNI` manifests.
+
+### Customizing Istio configuration
+
+The `spec.values` field of the `Istio` and `IstioCNI` resource can be used to customize Istio and Istio CNI plugin configuration using Istio's `Helm` configuration values.
+
+An example configuration:
+
+    ```yaml
+    apiVersion: sailoperator.io/v1alpha1
+    kind: Istio
+    metadata:
+    name: example
+    spec:
+    version: v1.23.0
+    values:
+        global:
+        mtls:
+            enabled: true
+        trustDomainAliases:
+        - example.net
+        meshConfig:
+        trustDomain: example.com
+        trustDomainAliases:
+        - example.net
+    ```
+
+For a list of available configuration for the `spec.values` field, run the following command:
+
+    ```sh
+    $ kubectl explain istio.spec.values
+    ```
+
+For the `IstioCNI` resource, replace `istio` with `istiocni` in the command above.
+
+Alternatively, refer to [Istio's artifacthub chart documentation](https://artifacthub.io/packages/search?org=istio&sort=relevance&page=1) for:
+
+- [Base parameters](https://artifacthub.io/packages/helm/istio-official/base?modal=values)
+- [Istiod parameters](https://artifacthub.io/packages/helm/istio-official/istiod?modal=values)
+- [Gateway parameters](https://artifacthub.io/packages/helm/istio-official/gateway?modal=values)
+- [CNI parameters](https://artifacthub.io/packages/helm/istio-official/cni?modal=values)
+- [ZTunnel parameters](https://artifacthub.io/packages/helm/istio-official/ztunnel?modal=values)
+
+## Installing the istioctl tool
```

Comment:
Since this content is similar to https://github.com/istio-ecosystem/sail-operator/tree/main/bundle#installing-the-istioctl-tool, it would be good if we avoid the duplication by pointing to the appropriate page (or re-arranging the content to its own page and linking them where necessary).

Reply:
That's a good point.

I made some re-arragement of the docs to avoid such duplication.